book/.gitignore

```diff
@@ -1,2 +1,3 @@
 *.html
+*.mustache
```

Comment:
why ignore the mustache files?

Reply:
Mustache file gets copied from ``ressources/template`` so I hardly see the point of comitting them in the book folder